cypress/support/commands.js

```diff
@@ -23,3 +23,29 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+const { convertShortMonthToFull } = require('./generate');
+
+Cypress.Commands.add('getAndTypeText', (selector, text) => {
+  cy.get(selector).type(text);
+});
+
+Cypress.Commands.add('chechTableDataBySelector', (selector) => {
+  cy.get(selector).then(function($value) {
+    const selectedValue = $value.text();
+    cy.get('td').should('contain.text', selectedValue);
+  });
+});
+
+Cypress.Commands.add('chechTableDataByValue', (value) => {
```

Comment:
```suggestion
Cypress.Commands.add('checkTableDataByValue', (value) => {
```

Reply:
Thank you for noticing these typos. Fixed.